test/dotnet.Tests/ParserTests/ToolRestoreParserTests.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.CommandLine;
+using Xunit;
+using Xunit.Abstractions;
+using Parser = Microsoft.DotNet.Cli.Parser;
+
+namespace Microsoft.DotNet.Tests.ParserTests
+{
+    public class ToolRestoreParserTests
+    {
+        private readonly ITestOutputHelper output;
+
+        public ToolRestoreParserTests(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        [Fact]
+        public void ToolRestoreParserCanGetManifestFilePath()
+        {
+            var command = Parser.Instance;
+            var result = command.Parse("dotnet tool restore folder/my-manifest.format");
```

Comment:
I don't think it makes sense to pass in the manifest file as a positional parameter.  I think probably you should have to pass something like `--toolManifest my-manifest.format`.

Reply:
fixed, changed to an option